Library/.rubocop.yml

```diff
@@ -63,26 +63,29 @@ Metrics/AbcSize:
   Max: 250
 
 Metrics/BlockLength:
-  Max: 1250
+  Enabled: false
```

Comment:
I'd like to keep these all enabled but make the maximum based on Homebrew/homebrew-core too.